senderSampleApp/src/main/java/org/kinecosystem/sampleSenderApp/kindiscover/SendKinService.java

```diff
@@ -36,14 +37,14 @@ KinTransferComplete transferKin(@NonNull String toAddress, int amount, @NonNull
             // here you may add some code to add the transaction details to
             // your app's transaction history metadata
 
-            return new KinTransferComplete(sourceAddress, transactionId.id(), transaction.getMemo());
+            callback.onSuccess(new KinTransferComplete(sourceAddress, transactionId.id(), transaction.getMemo()));
```

Comment:
Need to change the code in SendKinService to use the async version of build & send transaction !